src/Dscp/Crypto/Encrypt.hs

```diff
@@ -112,6 +112,10 @@ type CipherType = AES256
 authHeader :: ByteString
 authHeader = ""
 
+-- | Salt used for key derivation from password.
+passGenSalt :: ByteString
+passGenSalt = "dscp-educator-pass-gen"
```

Comment:
If you use fixed salt then adversary can still generate rainbow table and guess your password.
I may be wrong though. If I understand correctly, we used password hash as salt in similar code.

Reply:
No, in Cardano we've also used fixed salt for symmetric encryption key derivation (see `Pos.Crypto.HD`).
Generating a rainbow table for given key derivation function is kind of tiresome. The purpose of the _salt_ is to make your own key derivation function unique and different from other ones, thus preventing a rainbow table for your function from being _already available_.